app/src/docs/_examples/richTextEditor/Serialization.example.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import { Panel, FlexRow, Button, RichTextView } from '@epam/promo';
+import {
+    SlateEditor,
+    imagePlugin,
+    videoPlugin,
+    baseMarksPlugin,
+    linkPlugin,
+    iframePlugin,
+    quotePlugin,
+    superscriptPlugin,
+    headerPlugin,
+    listPlugin,
+    EditorValue,
+    codeBlockPlugin,
+    createSerializer,
+    isEditorValueEmpty,
+    migrateSchema,
+    createDeserializer,
+    paragraphPlugin,
+} from '@epam/uui-editor';
+import { demoData } from '@epam/uui-docs';
+import css from './SlateEditorBasicExample.module.scss';
+
+const plugins = [
+    paragraphPlugin(),
+    baseMarksPlugin(),
+    headerPlugin(),
+    superscriptPlugin(),
+    listPlugin(),
+    quotePlugin(),
+    linkPlugin(),
+    imagePlugin(),
+    videoPlugin(),
+    iframePlugin(),
+    codeBlockPlugin(),
+];
+
+const serializeHTML = createSerializer();
+const deserializeHTML = createDeserializer();
+
+export default function SlateEditorBasicExample() {
+    const [value, setValue] = useState<EditorValue>(migrateSchema(demoData.slateSerializationInitialData));
```

Comment:
Should we migrate state here, because we already have such migration inside `SlateEditor`? 
Anyway it will be better if we will replace our demo state with new state variant.

Reply:
Changed